### PR TITLE
tide: move repos into non-BZ flow

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -1119,13 +1112,20 @@ tide:
...
     - openshift/ci-ns-ttl-controller
     - openshift/ci-search
     - openshift/ci-secret-mirroring-controller
     - openshift/ci-tools
     - openshift/ci-vm-operator
+    - openshift/cincinnati
+    - openshift/cincinnati-graph-data
     - openshift/release
     - openshift/release-controller
+    - openshift/docker-distribution
```
Comment (wking) [inline core-services/prow/02_config/_config.yaml]: are these supposed to be alphabetized?  I guess not based on the previous content, but still a bit surprising.

**Comment by wking** [comment]
Needs a rebase (per GitHub, no label yet).

File: core-services/prow/02_config/_config.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply core-services/prow/02_config/_config.yaml]: no

**Pull request merged**